File: tests/human/x/fortran/map_membership.f90
program map_membership
  implicit none
  character(len=1) :: keys(2) = (/'a','b'/)
  logical :: hasA, hasC
  hasA = any(keys == 'a')
  hasC = any(keys == 'c')
  print *, hasA
  print *, hasC
end program map_membership

File: tests/human/x/pas/map_membership.pas
program MapMembership;

function HasKey(k: string): boolean;
begin
  if (k = 'a') or (k = 'b') then
    HasKey := True
  else
    HasKey := False;
end;

begin
  Writeln(HasKey('a'));
  Writeln(HasKey('c'));
end.
